### PR TITLE
[Synapse]-Update data plane monitoring swagger

### PR DESCRIPTION
**Pull request by sunsw1994**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by sunsw1994** [comment]
hi @jhendrixMSFT 

We can see all CI errors are about breaking change, and that change is by designed. We have updated  `workspaceName`-->`endpoint` in other data plane swagger files, so we also do the same update for this swagger file. 

Ping on reviewing this pull request. Thanks.

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2019-11-01-preview/monitoring.json [hunk 1]
```diff
@@ -180,162 +63,56 @@
...
         ],
         "responses": {
           "200": {
             "description": "Success",
             "schema": {
-              "$ref": "#/definitions/HistoryServerGraphResponse"
+              "$ref": "#/definitions/SqlQueryStringDataModel"
             }
           },
           "401": {
```
Comment (jhendrixMSFT) [inline specification/synapse/data-plane/Microsoft.Synapse/preview/2019-11-01-preview/monitoring.json:118]: I know this isn't part of the change, but having a 401 listed here like this indicates that SDKs should treat this as a *success*, is that really what you want?  If this response is intended to generate an error you need to use the [x-ms-error-response](https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-error-response) extension.

Comment (sunsw1994) [reply specification/synapse/data-plane/Microsoft.Synapse/preview/2019-11-01-preview/monitoring.json:118]: Thank you for your comments. And we think we need not do extra operations about 401, so we decide to remove that from responses.

**Comment by jhendrixMSFT** [comment]
The major breaking changes here is the removal of all the paths.  This is also by design?
Ping @idear1203 and @aim-for-better for their input.

**Comment by idear1203** [comment]
Hi @jhendrixMSFT yes, this is by design. This new Swagger spec is provided by the backend developers. Besides, @sunsw1994 already did some tests on these APIs and they work well.

**Comment by sunsw1994** [comment]
Yes, it is also by design. Let's wait for @idear1203 and @aim-for-better to double check it.

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**